postgresdb/seedUtils/copyRecords.sql

```diff
@@ -0,0 +1,6 @@
+TRUNCATE items CASCADE;
```

Comment:
Did you have any issues with the id not resetting to 1 after truncating a table? I had to use an ALTER SEQUENCE command whenever I deleted the table data.

Reply:
I don't have that issue. I specify the id in my CSV file so, that might be the reason.

Reply:
OK that makes sense. I used the auto-generated id for mine. Thanks for clearing it up.